src/common.cpp

```diff
@@ -113,5 +113,47 @@ instruction_ref add_common_op(module& m, const operation& op, std::vector<instru
     return insert_common_op(m, m.end(), op, std::move(inputs));
 }
 
+instruction_ref
+dot_apply_alpha_beta(module& m, const std::vector<instruction_ref>& args, float alpha, float beta)
+{
+    auto l1       = args[0];
+    auto l2       = args[1];
+    auto dot_type = l1->get_shape().type();
+    if(alpha != 1.0f)
```

Comment:
Use the float_equal function in the head file float_equal.hpp

Reply:
Done